docs/Network.md

```diff
@@ -50,6 +77,22 @@ fetch('https://mywebsite.com/endpoint.php')
   });
 ```
 
+2.  Called within an asynchronous function using ES7 `async`/`await` syntax:
```

Comment:
Is this transform enabled by default both on iOS and Android? If not people will find this example confusing because it won't work.